<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeMasterKeyWidget</class>
 <widget class="QWidget" name="ChangeMasterKeyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>153</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headlineLabel">
     <property name="text">
      <string>Headline</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="PasswordEditWidget" name="passwordEditWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KeyFileEditWidget" name="keyFileEditWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="YubiKeyEditWidget" name="yubiKeyEditWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PasswordEditWidget</class>
   <extends>QStackedWidget</extends>
   <header>gui/masterkey/PasswordEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KeyFileEditWidget</class>
   <extends>QStackedWidget</extends>
   <header>gui/masterkey/KeyFileEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>YubiKeyEditWidget</class>
   <extends>QStackedWidget</extends>
   <header>gui/masterkey/YubiKeyEditWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
